cmake/Dependencies.cmake

```diff
@@ -726,13 +726,15 @@ if(BUILD_PYTHON)
   endif()
 
   # Check that Python works
+  set(PYTHON_VERSION)
```

Comment:
Why do we need this line?

Reply:
> 
> 
> Why do we need this line?

In case NOT DEFINED PYTHON_EXECUTABLE,to define an empty variable